dashboard/api.py

```diff
@@ -367,3 +380,127 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
         # TODO: here goes the logic to pull the current grade  # pylint: disable=fixme
         pass
     return formatted_run
+
+
+def get_student_enrollments(user, edx_client):
```

Comment:
it is not that simple: for the certificate we pull the verified ones that are a subset of the certificates. I actually thought about turn these in calls to a generic function, but this generic function may turn to be pretty complicated.